src/Yaapii.Atoms/IO/TeeOutputStream.cs

```diff
@@ -80,8 +80,7 @@ public override void Flush()
             }
         }
 
-
-        public new void Dispose()
+        protected override void Dispose(bool disposing)
```

Comment:
Don't we need to override the Dispose() method also? I think now the base.Dispose() is called, which will not call this override. Or am i missing something?